typescript/src/entity.ts

```diff
@@ -37,4 +55,49 @@ export abstract class Entity {
   constructor() {
     this.propertiesMap = this.makePropertiesMap();
   }
+
+  public static get<T extends typeof Entity>(this: T, id: number):
+     Promise<InstanceType<T>>{
+    const resource = `/${this.resourceName}/${String(id)}/`;
+    return apiFetch(resource).then((data: any) => {
+      const result: InstanceType<T> = (new this()) as InstanceType<T>;
+      result.fromJson(data);
+      return result;
+    });
+  }
+
+  public static list<T extends typeof Entity>(this: T):
+      Promise<ListResponse<InstanceType<T>>> {
+    const resource = `/${this.resourceName}/`;
+    return apiFetch(resource).then((data: any) => {
+      const metadata = {canCreate: data.canCreate,
```

Comment:
![image](https://user-images.githubusercontent.com/2015244/63558018-71fdd800-c58e-11e9-8dcb-669495615d1c.png)

should we use subset pick

Reply:
the typescript types for lodash seem to be pretty good, so it can be used.

personally, i am not sure whether it really makes the code any easier to read in this case.